stdlib/public/core/Sequence.swift

```diff
@@ -470,6 +470,17 @@ public protocol Sequence {
   ///
   /// - Complexity: O(*n*), where *n* is the length of the sequence.
   func dropLast(_ n: Int) -> SubSequence
+  
+
+  /// Returns a subsequence by skipping elements while `predicate` returns
+  /// `true` and returning the remaining elements.
+  ///
+  /// - Parameter while: A closure that takes an element of the
+  ///   sequence as its argument and returns a Boolean value indicating
+  ///   whether the element is a match.
+  func drop(
+    while predicate: @noescape (Iterator.Element) throws -> Bool
+  ) rethrows -> SubSequence
```

Comment:
These two functions are new requirements on `Sequence`.  Please forward them in `AnySequence` wrappers (`ExistentialCollection.swift.gyb`).